preprocessors/action.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/action-recognition.schema.json",
+    "type": "object",
+    "title": "Human Action Recognition",
+    "description": "Preprocessor to detect human action in an image. The associated information can be found in preprocessor named 'ca.mcgill.a11y.image.preprocessor.actionrecognition'. ",
+    "properties":{
+        "actions":{
+            "type":"array",
+            "description": "Contains a list of actions detected by the preprocessor 'ca.mcgill.a11y.image.preprocessor.actionRecognition' or empty list if no actions are found",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "personID":{
+                        "description": "The ID of a person. The ID is generated by object detection preprocessor and is reused in this preprocessor",
+                        "type": "integer",
+                        "minimum": 0
+                    },
+                    "action":{
+                        "description": "Action label",
```

Comment:
It seems each label takes the form of gerund and underscore-separated details, so please make a note of that here and include a check that the label matches this convention.